test/integration/integration_test.exs

```diff
@@ -192,6 +192,28 @@ defmodule KafkaEx.Integration.Test do
     assert message.offset == offset
   end
 
+  test "fetch gets only new messages from kafka" do
+    random_string = generate_random_string
+    produce_response =  KafkaEx.produce(%Proto.Produce.Request{topic: random_string, required_acks: 1, messages: [%Proto.Produce.Message{value: "hey foo"}]}) |> hd
+    fetch_response = KafkaEx.fetch(random_string, 0) |> hd
+    message = fetch_response.partitions |> hd |> Map.get(:message_set) |> hd
+    assert message.value == "hey foo"
+    assert message.offset == 0
+
+    # There are no new messages, so fetch return no messages
+    fetch_response = KafkaEx.fetch(random_string, 0) |> hd
+    message_set = fetch_response.partitions |> hd |> Map.get(:message_set)
+    assert Enum.empty?(message_set)
+
+    # Produce a new message
+    produce_response =  KafkaEx.produce(%Proto.Produce.Request{topic: random_string, required_acks: 1, messages: [%Proto.Produce.Message{value: "bye foo"}]}) |> hd
+    # There's a new message, so fetch should return it
+    fetch_response = KafkaEx.fetch(random_string, 0) |> hd
+    message = fetch_response.partitions |> hd |> Map.get(:message_set) |> hd
+    assert message.value == "bye foo"
+    assert message.offset == 1
+  end
+
```

Comment:
this test should be in [`KafkaEx.ConsumerGroup.Test`](https://github.com/kafkaex/kafka_ex/blob/master/test/integration/consumer_group_test.exs) as offset_management is `consumer_group` related and not available in kafka < 0.8.2.

Also there are test cases that covers all this behavior (wrongly) that you should modify in [`KafkaEx.ConsumerGroup.Test`](https://github.com/kafkaex/kafka_ex/blob/master/test/integration/consumer_group_test.exs)